src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -13,7 +15,7 @@ const { actions, selectors } = annotations;
 
 function createTestStore() {
   return createStore(
-    [annotations, selection, drafts, groups, session, viewer],
+    [annotations, selection, defaults, drafts, groups, session, viewer],
```

Comment:
I know, I know, I know. We need to solve this `store` dependency/mocking strategy thing ASAP.

Reply:
This code smell is telling me that we should think about a more systematic pattern / rules for handling actions need to operate across multiple domains within the app or which are operations that essentially compose a set of low-level operations. At a minimum, I think we want to separate these operations into a separate module.

I've been having a think on this and will post some ideas later.

There is no need to change anything in this PR though.